packages/embed-grid/src/App.tsx

```diff
@@ -62,67 +62,50 @@ function App(): JSX.Element {
   const canCopy = searchParams.get('canCopy') != null;
   const canDownloadCsv = searchParams.get('canDownloadCsv') != null;
 
-  const initApp = useCallback(async () => {
-    try {
-      // Connect to the Web API server
-      const baseUrl = new URL(
-        process.env.REACT_APP_CORE_API_URL ?? '',
-        `${window.location}`
-      );
-
-      const websocketUrl = `${baseUrl.protocol}//${baseUrl.host}`;
+  useEffect(
+    function initializeApp() {
+      async function initApp() {
+        try {
+          // Get the table name from the query param `name`.
+          const name = searchParams.get('name');
 
-      log.debug(`Starting connection...`);
-      const connection = new dh.IdeConnection(websocketUrl);
+          if (!name) {
+            throw new Error('No name param provided');
+          }
 
-      log.debug('Getting console types...');
+          // Connect to the Web API server
+          const baseUrl = new URL(
+            process.env.REACT_APP_CORE_API_URL ?? '',
+            `${window.location}`
+          );
 
-      const types = await connection.getConsoleTypes();
+          const websocketUrl = `${baseUrl.protocol}//${baseUrl.host}`;
 
-      log.debug('Available types:', types);
+          log.debug(`Starting connection...`);
+          const connection = new dh.IdeConnection(websocketUrl);
```

Comment:
Not calling `SessionUtils.createConnection` here to avoid adding another dependency?

Reply:
Yes, SessionUtils is only in code-studio, so not importing to reduce dependencies.

Reply:
I could move it to jsapi-utils though I think. I'll do that.

Reply:
Actually nevermind - the createConnection basically just gets the websocket URL from env vars:
```
  const baseUrl = new URL(
    import.meta.env.VITE_CORE_API_URL ?? '',
    `${window.location}`
  );
```

We shouldn't do that in a lib or in jsapi-utils, so I will just keep them separate.